test/parallel/test-performance-eventlooputil.js

```diff
@@ -22,6 +22,12 @@ if (nodeTiming.loopStart === -1) {
                          { idle: 0, active: 0, utilization: 0 });
 }
 
+for (const p of ['name', 'entryType', 'startTime', 'duration',
+                 'nodeStart', 'v8Start', 'environment', 'loopStart', 'loopExit',
+                 'bootstrapComplete', 'idleTime'])
+  assert.ok(typeof JSON.parse(JSON.stringify(nodeTiming))[p] ===
+    typeof nodeTiming[p]);
+
```

Comment:
This test is very hard to read, could you consider adding variables to improve that?

```suggestion
{
  const jsonifiedTiming = JSON.parse(JSON.stringify(nodeTiming));
  const keys = ['name', 'entryType', 'startTime', 'duration',
                'nodeStart', 'v8Start', 'environment', 'loopStart', 'loopExit',
                'bootstrapComplete', 'idleTime'];
  for (const p of keys)
    assert.ok(typeof jsonifiedTiming[p] === typeof nodeTiming[p]);
}

```